agrona/src/main/java/org/agrona/concurrent/ringbuffer/ManyToOneRingBuffer.java

```diff
@@ -29,6 +32,10 @@
  */
 public final class ManyToOneRingBuffer implements RingBuffer
 {
+    /**
+     * Minimal required capacity of the ring buffer excluding {@link RingBufferDescriptor#TRAILER_LENGTH}.
+     */
+    static final int MIN_CAPACITY = HEADER_LENGTH;
```

Comment:
This should be public if referenced in Javadoc.

Reply:
Done.